front-end/src/main.js

```diff
@@ -1,5 +1,7 @@
 import { createApp } from 'vue'
 import App from './App.vue'
 import router from './router'
+import '@fortawesome/fontawesome-free/css/all.css'
+import '@fortawesome/fontawesome-free/js/all.js'
```

Comment:
J'ai des problèmes avec l'installation de les dependencies:

@fortawesome/fontawesome-free/css/all.css in ./src/main.js
@fortawesome/fontawesome-free/js/all.js in ./src/main.js
Est-ce que tu peux m'expliquer comment tu as les installé quand tu as du temps ?

Reply:
J'ai resolu ça avec:
`npm i --save @fortawesome/vue-fontawesome@prerelease`